cmd/deploy.go

```diff
@@ -63,7 +63,7 @@ that is pushed to an image registry, and finally the function's Knative service
 	// Flags shared with Build specifically related to building:
 	cmd.Flags().StringP("builder", "", "pack", "build strategy to use when creating the underlying image. Currently supported build strategies are 'pack' and 's2i'.")
 	cmd.Flags().StringP("builder-image", "", "", "builder image, either an as a an image name or a mapping name.\nSpecified value is stored in func.yaml (as 'builder' field) for subsequent builds. ($FUNC_BUILDER_IMAGE)")
-	cmd.Flags().StringP("image", "i", "", "Full image name in the form [registry]/[namespace]/[name]:[tag] (optional). This option takes precedence over --registry (Env: $FUNC_IMAGE)")
+	cmd.Flags().StringP("image", "i", "", "Full image name in the form [registry]/[namespace]/[name]:[tag]@[digest] (optional). This option takes precedence over --registry (Env: $FUNC_IMAGE). If digest is given, build and push are disabled.")
```

Comment:
Just a nit, I'd highlight that digest is not mandatory and also put the `(Env: $FUNC_IMAGE)` to the end of the section.
```suggestion
	cmd.Flags().StringP("image", "i", "", "Full image name in the form [registry]/[namespace]/[name]:[tag]@[digest] (optional). This option takes precedence over --registry. Specifying digest in the full image name is not mandatory, but if it is given, 'build' and 'push' phases are disabled. (Env: $FUNC_IMAGE)")
```

Something like this^^